core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -209,6 +210,36 @@ class ZkUtils(val zkClient: ZkClient,
     }
   }
 
+  /* Represents a cluster identifier. Stored in Zookeeper in JSON format: {"version" -> "1", "id" -> id } */
+  object ClusterIdentifier {
```

Comment:
Should probably just call it `ClusterId` as that is the name we've been using for some time.